WordPress/Sniffs/PHP/NoUnneededTernarySniff.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Sniffs\PHP;
+
+use WordPressCS\WordPress\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Discourage the usage of the unneeded ternary operator.
+ *
+ * This sniff is influenced by the same rule set in the ESLint rules.
+ * When we want to store a boolean value in a variable, using a ternary operator
```

Comment:
Should this sniff also cover `return ... ? true : false;`?

Reply:
I think this is the point of the sniff. Wherever there is an unneeded ternary detected, it can be removed. This includes conditionals, return statements or assignment to a variable 👍

Reply:
And yet, there are no unit tests which shows that return statements are covered by this sniff.

Reply:
Will add those when I update the sniff :+1: